Common/cpp/Tools/JSISerializer.cpp

```diff
@@ -53,39 +51,38 @@ std::string stringifyJSIFunction(jsi::Runtime &rt, const jsi::Function &func) {
 std::string stringifyJSIHostObject(
     jsi::Runtime &rt,
     jsi::HostObject &hostObject) {
-  std::stringstream ss;
   int status = -1;
   const char *hostObjClassName =
```

Comment:
Do we need to call `free(hostObjClassName)`?

Reply:
you're right, @t0ms0n00  please wrap it with unique_ptr with custom deleter:
```
std::unique_ptr<char, decltype(&free)> hostObjClassName(abi::__cxa_demangle(typeid(hostObject).name(), NULL, NULL, &status), &free);
```